src/browzine-summon-adapter.js

```diff
@@ -198,34 +198,225 @@ browzine.search = (function() {
   };
 }());
 
+browzine.serSol360Core = (function() {
+  var api = browzine.api;
+  var apiKey = browzine.apiKey;
+
+  function getQueryVariable(url, key) {
+    var query = url.split("?")[1];
+    var parameters = query.split("&");
+    var value = "";
+
+    for(var i = 0; i < parameters.length; i++) {
+      var pair = parameters[i].split("=");
+
+      if(pair[0] == key) {
+        value = pair[1];
+        return value;
+      }
+    }
+
+    return value;
+  }
+
+  function getIssn(title) {
+    var issn = "";
+
+    //Using the base image url, if an issn exists, then the issn query parameter will be the issn.
+    //However, if only an eissn exists, then the issn query parameter value will be the eissn.
+    //This way we're able to account for both issn and eissn journal identifiers.
+    //e.g. "History matters (Boone, N.C.)"
+    if(title.syndeticsImageUrl) {
+      issn = getQueryVariable(title.syndeticsImageUrl, "issn");
+    }
+
+    return encodeURIComponent(issn);
+  };
+
+  function getTarget(issn) {
+    var element = $(".results-identifier:contains('" + issn + "')").closest(".results-title-row");
+    var target = element.length > 0 ? element[0] : undefined;
+
+    return target;
+  };
+
+  function getEndpoint(issn) {
+    var endpoint;
+
+    if(issn) {
+      endpoint = api + "/search?issns=" + issn.trim().replace("-", "");
+      endpoint += "&access_token=" + apiKey;
+    }
+
+    return endpoint;
```

Comment:
Just checking this method because we declare `endpoint` without setting it. Are we OK returning `undefined` here if an issn isn't passed in? Some similar methods below (like `getCoverImageUrl`) seem to set the value as null first, so I figured I'd double check the outlier.

Reply:
Consistency is good. Thanks!